docs/HowTo/Monitor/Metrics.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Monitoring and metrics
+---
+
+Enable the [Prometheus](https://prometheus.io/) monitoring and alerting service for
+Teku metrics using the [`--metrics-enabled`](../../Reference/CLI/CLI-Syntax.md#metrics-enabled)
+option.
+
+## Install Prometheus
+
+To use Prometheus with Teku, install the
+[Prometheus main component](https://prometheus.io/download/). On MacOS, install with
+[Homebrew](https://formulae.brew.sh/formula/prometheus):
+
+ ```bash
+ brew install prometheus
+```
+
+!!! tip
+
+    You can also install:
+
+    * [Exporters] to monitor non-Teku specific items such as disk usage and CPU usage.
+    * Other Prometheus components, such as the Alert Manager. Additional configuration is not
+      required for these components because Prometheus handles and analyzes data directly from the
+      feed.
```

Comment:
do we need to doc this here or can ppl just read about it at Prometheus website?

Reply:
True, we probably don't need to teach them how to use prometheus.